test/runtests.jl

```diff
@@ -533,7 +535,90 @@ end
     x = [3; 2; 1]
     y = zeros(3)
     @test quantile!(y, x, [0.1, 0.5, 0.9]) === y
-    @test y == [1.2, 2.0, 2.8]
+    @test y ≈ [1.2, 2.0, 2.8]
+
+    #tests for quantile calculation with configurable α and β parameters
+    v = [2, 3, 4, 6, 9, 2, 6, 2, 21, 17]
+
+    # tests against SciPy quantile method
+    @test quantile(v, 0.0, α=0.0, β=0.0) ≈ 2.0
+    @test quantile(v, 0.2, α=1.0, β=1.0) ≈ 2.0
+    @test quantile(v, 0.4, α=0.0, β=0.0) ≈ 3.4000000000000004
```

Comment:
```suggestion
    @test quantile(v, 0.4, α=0.0, β=0.0) ≈ 3.40
```
(etc., if the that precision is not needed)

Reply:
This is a strange effect resulting from automatic generation of test cases. I rounded them to 2 digits, the test cases are still successful, but look nicer. Thanks for the feedback!